logging/bin/deploy_kibana_content.sh

```diff
@@ -0,0 +1,187 @@
+#! /bin/bash
+
+# Copyright © 2020, SAS Institute Inc., Cary, NC, USA.  All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+cd "$(dirname $BASH_SOURCE)/../.."
+source logging/bin/common.sh
+source logging/bin/secrets-include.sh
+
+this_script=`basename "$0"`
+
+log_debug "Script [$this_script] has started [$(date)]"
+
+KIBANA_CONTENT_DEPLOY=${KIBANA_CONTENT_DEPLOY:-${ELASTICSEARCH_ENABLE:-true}}
+
+if [ "$KIBANA_CONTENT_DEPLOY" != "true" ]; then
+  log_info "Environment variable [KIBANA_CONTENT_DEPLOY] is not set to 'true'; exiting WITHOUT deploying content into Kibana"
+  exit 0
+fi
+
+# temp file used to capture command output
+tmpfile=$TMP_DIR/output.txt
+
+# Confirm namespace exists
+if [ "$(kubectl get ns $LOG_NS -o name 2>/dev/null)" == "" ]; then
+  log_error "Namespace [$LOG_NS] does NOT exist."
+  exit 1
+fi
+
+# Require TLS into Kibana?
+LOG_KB_TLS_ENABLE=${LOG_KB_TLS_ENABLE:-false}
+
+if [ "$LOG_KB_TLS_ENABLE" == "true" ]; then
+   # w/TLS: use HTTPS in curl commands
+   KB_CURL_PROTOCOL=https
+   log_debug "TLS enabled for Kibana"
+else
+   # w/o TLS: use HTTP in curl commands
+   KB_CURL_PROTOCOL=http
+   log_debug "TLS not enabled for Kibana"
+fi
+
+# get credentials
+get_credentials_from_secret admin
+rc=$?
+if [ "$rc" != "0" ] ;then log_info "RC=$rc"; exit $rc;fi
+
+set -e
+
+
+log_info "Configuring Kibana"
+
+#### TEMP?  Remove when defining nodePort via HELM chart?
```

Comment:
This isn't a question - when the helm chart supports nodePort, we'd move to using that.